content/contributing/opencue/Getting started/getting-started-linux.md

```diff
@@ -157,9 +157,13 @@ To build and run it with IntelliJ IDEA:
    IntelliJ downloads Gradle and all source dependencies, then compiles the project and
    runs tests.
 
-1. Browse to the `src/main/java/com.imageworks/spcue/CuebotApplication` file.
+1. To setup run configurations go to **Run** > **Edit Configurations**
 
-1. Click **Edit 'CubotApplicat....main()'...**.
+1. Click the **+** icon on the top left corner to add a new configuration. Click **Application** on the drop down as shown in the screenshot below.
+
+   ![A screenshot of IntelliJ add config list](/docs/images/windows/cuebot_intellij_add_config.png)
```

Comment:
Looks like the images are broken in the preview: https://deploy-preview-277--elated-haibt-1b47ff.netlify.app/contributing/opencue/getting-started/getting-started-linux/

I think you need to remove the `windows/` from the path here. he images are stored directly in `content/docs/images/`, which IMO is the correct place for them.

Reply:
Updated in https://github.com/AcademySoftwareFoundation/opencue.io/pull/277/commits/e1fbea93b3865660350cf62098dda45c5ea60cba

> I think you need to remove the windows/ from the path here.

Noted.
Since there's an issue with my Hugo installation, I couldn't check whether the images worked in the deployed version so took the paths given by other image references on the same page and applied it to the new ones. 
https://github.com/AcademySoftwareFoundation/opencue.io/blob/2b20ec4b2513293a8059c795be1e5043da9ed35c/content/contributing/opencue/Getting%20started/getting-started-linux.md?plain=1#L329